packages/client/src/features/change-bounds/di.config.ts

```diff
@@ -0,0 +1,25 @@
+/********************************************************************************
+ * Copyright (c) 2019-2023 EclipseSource and others.
```

Comment:
```suggestion
 * Copyright (c) 2023 EclipseSource and others.
```

Please have a look at all the copyright headers and coordinate with Haydar on the correct copyright holder that should be applied.